lib/kernel/test/interactive_shell_SUITE.erl

```diff
@@ -987,6 +989,40 @@ shell_receive_standard_out(Config) ->
         ok
     after
         stop_tty(Term)
+    end,
+    Term2 = start_tty([{args,["-stdlib","shell_redraw_prompt_on_output","false"]}|Config]),
+    try
+        send_tty(Term2,"my_fun(5) -> ok; my_fun(N) -> receive after 100 -> io:format(\"~p\\n\", [N]), my_fun(N+1) end.\n"),
+        send_tty(Term2,"spawn(shell_default, my_fun, [0]). ABC\n"),
+        timer:sleep(1000),
+        check_location(Term2, {0,-18}), %% Check that we are at the same location relative to the start.
+        check_content(Term2, "..0\\s+1\\s+2\\s+3\\s+4"),
+        ok
+    after
+        stop_tty(Term2)
+    end.
+shell_receive_user_output(Config) ->
```

Comment:
Does this do anything different from the previous test?

Reply:
Yes, it prints to `user` instead of `standard_io`, which means that it will not go via `group:more_data` but directly to `prim_tty`.

Reply:
Ah I see